### PR TITLE
Preparing release 0.6.0

### PR DESCRIPTION
**Pull request by Natim**
r? @ametaireau

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,10 +4,11 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-0.6.0 (unreleased)
+0.6.0 (2015-10-12)
 ==================
 
-- Nothing changed yet.
+- Supports requests-hawk 0.2.0 (#17)
```
Comment (almet) [inline CHANGELOG.rst]: Update requests-hawk to 0.2.0*

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,10 +4,11 @@ CHANGELOG
...
+- Add support for oldest sort keyword. (#18)
```
Comment (almet) [inline CHANGELOG.rst]: Add support for sorting by oldest when retrieving records*

File: CHANGELOG.rst (hunk 1 above)
Comment (Natim) [inline CHANGELOG.rst]: That's not what I meant, it is more `Supports requests-hawk 0.2.0 drawback`

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,10 +4,11 @@ CHANGELOG
 This document describes changes between each past release.
 
 
-0.6.0 (unreleased)
+0.6.0 (2015-10-12)
 ==================
 
-- Nothing changed yet.
+- Supports requests-hawk 0.2.0 drawbacks. (#17)
```
Comment (almet) [inline CHANGELOG.rst]: You need to be consistent. I would suggest to use the infinitive of the verb (support, add) everywhere.

Comment (almet) [inline CHANGELOG.rst]: Ah, I get it now. I would say "Circumvent a limitation present in requests-hawk 0.2.0"

**Pull request merged**